src/components/DevTools/DevTools.tsx

```diff
@@ -0,0 +1,117 @@
+import { useMemo, useState } from 'react';
```

Comment:
It probably makes sense to combine this file with the logic in overrideConfig, since they overlap almost entirely in function and purpose